static/app/components/feedback/feedbackItem/feedbackItemLoader.tsx

```diff
@@ -37,9 +38,13 @@ export default function FeedbackItemLoader() {
     <Placeholder height="100%" />
   ) : issueResult.isError ? (
     <FeedbackErrorDetails error={t('Unable to load feedback')} />
-  ) : !issueData ? (
-    <FeedbackEmptyDetails />
+  ) : issueData ? (
+    <ErrorBoundary
+      customComponent={<FeedbackErrorDetails error={t('Unable to load feedback')} />}
+    >
+      <FeedbackItem eventData={eventData} feedbackItem={issueData} tags={tags} />
+    </ErrorBoundary>
   ) : (
-    <FeedbackItem eventData={eventData} feedbackItem={issueData} tags={tags} />
+    <FeedbackEmptyDetails />
```

Comment:
inverted the conditional here: `!issueData` becomes `issueData` and the result is flipped to match